core/core/src/main/scala/biz/lobachev/annette/core/utils/ChimneyCommons.scala

```diff
@@ -0,0 +1,7 @@
+package biz.lobachev.annette.core.utils
+
+object ChimneyCommons {
+
+  implicit val projectConfig =
+    io.scalaland.chimney.dsl.TransformerConfiguration.default.enableMethodAccessors.enableDefaultValues
```

Comment:
Sets up the same defaults as in 0.6.1 (which community considered unsafe and thus requiring opt-in).